v22.2/known-limitations.md

```diff
@@ -10,6 +10,85 @@ This page describes newly identified limitations in the CockroachDB {{page.relea
 
 ## New limitations
 
+### DROP OWNED BY does not support drop functions
+
+`DROP OWNED BY` drops all owned objects as well as any grants on objects not owned by the role.
+
+In its current implementation, this statement does not drop functions. Users must drop their functions manually.
+
+[Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/90476)
+
+{% comment %}
+    initial DROP OWNED BY implementation PR/issue:
+    https://github.com/cockroachdb/cockroach/pull/82936
+    https://github.com/cockroachdb/cockroach/issues/55381
+    doc work in progress (to link to after merge):
+    https://github.com/cockroachdb/docs/pull/15094/files
+    https://cockroachlabs.atlassian.net/browse/DOC-4770 
+    review: rafiss (or devadvocado)
+{% endcomment %}
+
+### DROP OWNED BY not supported where role has synthetic privileges
+
+`DROP OWNED BY` drops all owned objects as well as any grants on objects not owned by the role.
+
+In its current implentation, this operation cannot be performed for roles that have synthetic privileges (entries in `system.privileges`).
+
+Instead, you can use `REVOKE SYSTEM` for the relevant privileges the role has in `system.privileges`.
+
+[Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/88149)
+
+{% comment %}
+    initial DROP OWNED BY implementation PR/issue:
+    https://github.com/cockroachdb/cockroach/pull/82936
+    https://github.com/cockroachdb/cockroach/issues/55381
+    doc work in progress (to link to after merge):
+    https://github.com/cockroachdb/docs/pull/15094/files
+    https://cockroachlabs.atlassian.net/browse/DOC-4770 
+    review: rafiss
+{% endcomment %}
+
+### Expressions with `*` not supported in user defined functions (UDFs)
+
+Expressions such as `SELECT *` are not currently allowed within the body of a UDF.
+
+[Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/90080)
```

Comment:
@mgartner Does this heading and description work as-is? It's a bit brief; I'd welcome suggestions for additional detail, if applicable.